### ISSUE TITLE
A11y_VSTS Wiki (4 Modules)_Embed Azure Boards results_Screen Reader: User is unable to navigate into the cells using ctrl+alt+rt arrow when NVDA is ON

### ISSUE THREAD

**Issue by v-sanana** [v-sanana/sampath#1]
**Environment Details:**
Application URL: acctesting.visualstudio.com​
Browser: Chrome
Screen Reader: NVDA
​
**Repro Steps:​**
1)Hit the URL https://acctesting.visualstudio.com/​
2)Select any of the project and then navigate to "Wiki" on the left navigation bar​
3)Navigate to "New Page" and press enter​
4)Navigate to "more Options" control and select "Query results" list item​
5)Insert any Shared queries​
6)Tab till Load Results and Press Enter​
7)Start NVDA
8)Tab till the list items present in the grid 
9)Verify user is able to navigate through the cells using Ctrl+alt+rt arrow 

​
**Actual Result: ​**
On pressing Ctrl+alt+rt arrow NVDA is announcing "Not in a table cell". user is unable to navigate  through the cells using Ctrl+alt+rt arrow. The table is not defined properly.

**Expected Result:​**
On pressing Ctrl+alt+rt arrow NVDA should read the list items present in the cell along with the associated column headers. 
​
**User Impact:​**
Visual Impaired Users will face difficulties in navigating through the cells if they will not be able to navigate into the cells using shortcut keys
​
**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

[Bug_1475584_User is unable to navigate cell by cell using ctrl+alt+rt arrow(3).pptx](https://github.com/v-sanana/sampath/files/3652435/Bug_1475584_User.is.unable.to.navigate.cell.by.cell.using.ctrl%2Balt%2Brt.arrow.3.pptx)
[Bug_1475584_User is unable to navigate cell by cell using ctrl+alt+rt arrow(1).pptx](https://github.com/v-sanana/sampath/files/3652436/Bug_1475584_User.is.unable.to.navigate.cell.by.cell.using.ctrl%2Balt%2Brt.arrow.1.pptx)
[Bug_1475584_User is unable to navigate cell by cell using ctrl+alt+rt arrow(2).pptx](https://github.com/v-sanana/sampath/files/3652437/Bug_1475584_User.is.unable.to.navigate.cell.by.cell.using.ctrl%2Balt%2Brt.arrow.2.pptx)
